### PR TITLE
Playtest changes and cleanup

### PR DESCRIPTION
**Pull request by triggrhappy0**

### CODE REVIEW

File: pkto_cockatrice.xml [hunk 1]
```diff
@@ -1386,7 +1386,7 @@ When Hitmonchan enters the battlefield, the next time a source of your choice wo
 			<type>Pokémon - Normal</type>
 			<pt>3/7</pt>
 			<tablerow>2</tablerow>
-			<text>When Lickitung enters the battlefield, target Pokémon can't attack, block, or activate abilities until Lickitung leaves the battlefield.</text>
+			<text>When Lickitung enters the battlefield, until Lickitung leaves the battlefield, target Pokémon can’t attack or block and its activated abilities can’t be activated.</text>
```
Comment (brooks42) [inline pkto_cockatrice.xml:1389]: weird ' in "can't"

File: pkto_cockatrice.xml [hunk 2]
```diff
@@ -2364,7 +2364,7 @@ Draw a card.</text>
 			<manacost>1{U/B}</manacost>
 			<type>Instant</type>
 			<tablerow>3</tablerow>
-			<text>Counter target spell. If that spell is countered this way, put it into its owner's hand instead of into that player's graveyard. Spells with the same name as that spell can't be cast until your next turn.</text>
+			<text>Counter target spell. If that spell is countered this way, put it into its owner’s hand instead of into that player’s graveyard. That player can’t cast spells with the same name as that spell until the end of their next turn.</text>
```
Comment (brooks42) [inline pkto_cockatrice.xml:2367]: weird ' again

**Comment by brooks42** [comment]
Agatha says "on up to on":

image

**Review by brooks42 — CHANGES_REQUESTED** [review]
overall good changes but some small tweaks before we can merge

File: pkto_cockatrice.xml (hunk 1 above)
Comment (triggrhappy0) [reply pkto_cockatrice.xml:1389]: Friggin MSE. Fixed.

**Pull request merged**